presto-session-property-managers/src/main/java/com/facebook/presto/session/SessionMatchSpec.java

```diff
@@ -99,6 +108,17 @@ public Map<String, String> match(SessionConfigurationContext context)
             }
         }
 
+        if (maxVersion.isPresent() || minVersion.isPresent()) {
+            boolean validVersion = true;
+            if (maxVersion.isPresent()) {
+                validVersion = coordinatorVersion.lessThanOrEqualTo(maxVersion.get());
+            }
+            if (minVersion.isPresent()) {
+                validVersion = validVersion && coordinatorVersion.greaterThanOrEqualTo(minVersion.get());
+            }
+            return validVersion ? sessionProperties : ImmutableMap.of();
```

Comment:
can we change that to:
```
if (!validVersion) {
    return ImmutableMap.of();
}
```
This will make sure we return sessionProperties only from one place.